src/pkg/cli/client/byoc.go

```diff
@@ -340,6 +341,21 @@ func (b *byocAws) runCdCommand(ctx context.Context, cmd ...string) (awsecs.TaskA
 		"PULUMI_CONFIG_PASSPHRASE": pkg.Getenv("PULUMI_CONFIG_PASSPHRASE", "asdf"),                          // TODO: make customizable
 		"STACK":                    b.pulumiStack,
 	}
+}
+
+func (b *byocAws) runCommand(ctx context.Context, cmd ...string) (awsecs.TaskArn, error) {
+	command := exec.CommandContext(ctx, cmd[0], cmd[1:]...)
+	command.Stdout = os.Stdout
+	command.Stderr = os.Stderr
+	command.Env = os.Environ()
+	for k, v := range b.environment() {
+		command.Env = append(command.Env, k+"="+v)
+	}
+	return nil, command.Start()
```

Comment:
Is it possible that this may result in same environment variable appear twice? If it is set both in the os env and the code above? Why do we want to have the os environment variables passed down to CD? this could mean the AWS keys being sent over to CD image right?